CLAUDE.md

```
@@ -0,0 +1,57 @@
+just lint               # Run golangci-lint
+just check              # Run fmt, vet, and test
+just run --help         # Build and run with arguments
+```
+
+## Architecture
+
+This is a CLI tool that orchestrates Claude CLI to run automated development workflows. It spawns Claude as a subprocess, parses its streaming JSON output, and displays formatted results.
+
+### Package Dependencies
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)